resources/views/pages/auth/login.blade.php

```diff
@@ -188,10 +188,12 @@ public function authenticate()
             </form>
 
 
-            <div class="mt-3 space-x-0.5 text-sm leading-5 text-left" style="color:{{ config('devdojo.auth.appearance.color.text') }}">
-                <span class="opacity-[47%]"> {{ config('devdojo.auth.language.login.dont_have_an_account') }} </span>
-                <x-auth::elements.text-link data-auth="register-link" href="{{ route('auth.register') }}">{{ config('devdojo.auth.language.login.sign_up') }}</x-auth::elements.text-link>
-            </div>
+            @if(config('devdojo.auth.settings.registration_enabled'))
```

Comment:
We may want to add a default value of true in this config, like so:

```php
@if(config('devdojo.auth.settings.registration_enabled', true))
```

Reply:
Good point, just updated that!